tests/django_test_setup.py

```diff
@@ -22,6 +22,11 @@ def setup_test_config(
                     "tests/templates/",
                     "tests/components/",  # Required for template relative imports in tests
                 ],
+                "OPTIONS": {
+                    "builtins": [
+                        "django_components.templatetags.component_tags",
```

Comment:
This makes it so that we don't have to call `{% load component_tags %}` in tests. See https://github.com/EmilStenstrom/django-components/discussions/900#discussioncomment-11834328